RNTester/js/examples/KeyboardEventsExample/KeyboardEventsExample.js

```diff
@@ -51,12 +55,16 @@ class KeyEventExample extends React.Component<{}, State> {
             <View
               acceptsKeyboardFocus={true}
               enableFocusRing={true}
+              validKeysDown={["a", "b", "c"]}
               onKeyDown={this.onKeyDownEvent}
+              validKeysDown={["d", "e", "f"]}
```

Comment:
Are these meant to be duplicated? Or is the second one supposed to be validKeysUp? Separately, can we have one of the tests use the arrow keys as an example, since I expect that would be a common use case?

Reply:
They don't need to be duplicated. I used different ones on purpose

Reply:
So line 60 is not supposed to be validKeysUp instead of validKeysDown?

Reply:
Oh that's what you mean- I thought you were saying the chars should be the same. Yea it should be up